io.catenax.quality.parquet_exchange/1.0.0/ParquetExchange.ttl

```diff
@@ -0,0 +1,82 @@
+#######################################################################
+# Copyright (c) 2025 Robert Bosch GmbH
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix :<urn:samm:io.catenax.quality.parquet_exchange:1.0.0#> .
+
+:ParquetExchange a samm:Aspect ;
+   samm:preferredName "Parquet Exchange"@en ;
+   samm:description "Aspect model for parquet file representation."@en ;
+   samm:properties ( :mappingRule :content :dataParquetFile ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:mappingRule a samm:Property ;
+   samm:preferredName "Mapping rule"@en ;
+   samm:description "Mapping rule used for reading parquet file"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "CX-0123" .
+
+:content a samm:Property ;
+   samm:preferredName "Content"@en ;
+   samm:description "Content of aspect model defining the usecase"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "io.catenax.fleet.diagnostic_data" .
+
+:dataParquetFile a samm:Property ;
+   samm:preferredName "Data of parquet file"@en ;
+   samm:description "This property refers to the information particular to parquet file"@en ;
+   samm:characteristic :FileCharacteristic .
+
+:FileCharacteristic a samm:Characteristic ;
+   samm:preferredName "File Characteristic"@en ;
+   samm:description "Characteristic for handling parquet file"@en ;
+   samm:dataType :ParquetFileDataEntity .
+
+:ParquetFileDataEntity a samm:Entity ;
+   samm:preferredName "Parquet file data entity"@en ;
+   samm:description "Entity object for handling parquet file data, example edc assetid, dataplane and controlplane url."@en ;
+   samm:properties ( :dataplaneUrl :encodingType :controlplaneUrl :edcAssetId ) .
```

Comment:
do not use "edc" in names, EDC is an implementation, the product is named DCP (dataspace protocol), so perhaps dcpAssetId. Probably add prefix to all properties?